pkg/middlewares/loopstop/loop_stop.go

```diff
@@ -0,0 +1,63 @@
+package loopstop
+
+import (
+	"crypto/sha1"
+	"fmt"
+	"net/http"
+
+	"github.com/containous/alice"
+	"github.com/traefik/traefik/v2/pkg/log"
+)
+
+const xTraefikRouter = "X-Traefik-Router"
```

Comment:
```suggestion
const headerXTraefikRouter = "X-Traefik-Router"
```

Reply:
We have used the same naming convention as in the Forwarded middleware.